instrumentation/vertx-reactive-3.5/javaagent/src/latestDepTest/java/VertxReactiveWebServer.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import static io.opentelemetry.instrumentation.test.base.HttpServerTest.ServerEndpoint.SUCCESS;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.reactivex.Single;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.reactivex.core.AbstractVerticle;
+import io.vertx.reactivex.core.Vertx;
+import io.vertx.reactivex.core.http.HttpServerResponse;
+import io.vertx.reactivex.ext.jdbc.JDBCClient;
+import io.vertx.reactivex.ext.sql.SQLConnection;
+import io.vertx.reactivex.ext.web.Router;
+import io.vertx.reactivex.ext.web.RoutingContext;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class VertxReactiveWebServer extends AbstractVerticle {
+
+  private static final Logger logger = LoggerFactory.getLogger(VertxReactiveWebServer.class);
+
+  private static final Tracer tracer = GlobalOpenTelemetry.getTracer("test");
+
+  public static final String TEST_REQUEST_ID_PARAMETER = "test-request-id";
+  public static final String TEST_REQUEST_ID_ATTRIBUTE = "test.request.id";
+
+  private static final String CONFIG_HTTP_SERVER_PORT = "http.server.port";
+  private static JDBCClient client;
```

Comment:
Why do we need JDBC/DB in http server test?

Reply:
Presumably because `JDBCClient` is also reactive. Anyway all of this is a slightly modified copy of 3.5 test.